events/equeue/tests/tests.c

```diff
@@ -687,6 +707,45 @@ void multithreaded_barrage_test(int N) {
     equeue_destroy(&q);
 }
 
+struct sCaQ
+{
+    int p;
+    equeue_t* q;
+};
+
+typedef struct sCaQ CountAndQueue;
```

Comment:
nit: This is inconsistent with other structs in this file. Could you change this to `struct count_and_queue` without a typedef.